docs/content/learn/tutorials/control/basic-profile.md

```diff
@@ -0,0 +1,1277 @@
+---
+title: Creating a Profile
+description: A tutorial on creating a Baseline using an OSCAL Profile.
+weight: 20
+suppresstopiclist: true
+toc:
+  enabled: true
+aliases:
+  - /tutorials/profile/
+  - /learn/tutorials/profile/
+---
+
+This tutorial covers creating and modifying a subset of controls from a catalog in OSCAL by using the OSCAL profile model. Such subset is often refer to as basic baseline. 
+
+Before reading this tutorial you should:
+- Have some familiarity with the [XML](https://www.w3.org/standards/xml/core), [JSON](https://www.json.org/), or [YAML](https://yaml.org/spec/) formats.
+- Review the OSCAL control layer [overview](/concepts/layer/control/).
+- Review the OSCAL [catalog](/concepts/layer/control/catalog/) and [profile](/concepts/layer/control/profile/) model overview pages.
+- Review the tutorial on [Creating a Control Catalog](/learn/tutorials/catalog/).
+
+## What is an OSCAL Profile?
+
+A [baseline][baseline-definition], or overlay in other terminology, defines a specific set of selected security control requirements from one or more control catalogs for use in managing risks in an information system.
+
+In the most basic sense, a baseline is simply another catalog, and could be represented using the OSCAL Catalog Model. However, baselines are fundementally based on and rooted in whichever source catalog(s) they use as a foundation. With this in mind, OSCAL uses a different model to represent baselines that identifies the source catalog(s),which controls it will "select" or "import", and any changes made to those controls. An **OSCAL Profile** is a machine-readable representation of a **baseline**, expressed using the OSCAL [Profile model][profile-docs], which includes contextualizing documentation and metadata.
+
+OSCAL Profiles are created within the context of the [Profile Resolution Specification](/concepts/processing/profile-resolution/), which defines exactly how tools can programmatically understand OSCAL Profiles. 
```

Comment:
The word 'Profile' is sometimes capitalized, some other times it is not. We need consistency and I suggest we use lower case.

Reply:
I've corrected the capitalization of terms throughout the document.

Also the profile resolution blurb has been rewritten so that it highlights that this is an optional operation.